modules/ROOT/pages/endpoints.adoc

```diff
@@ -71,4 +71,18 @@ Select the section that corresponds to the Protocol-Type of the Endpoint for whi
 
 * xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
-=======
+// ====
+
+== Delete Endpoints
+
+From either the Partner or Host page, you can delete any endpoints that are not used in message flow configurations.
+
+. From within your Anypoint Partner Manager Sandbox or Production environment, select Partner or <host partner name> from the upper left.
+
+. Search for the endpoint you want to delete by Name, Protocol, URL, or Description.
++
+You can perform a search from within _Endpoints: Receive from partners_, _Endpoints: Targets at <partner name>_ for the Host, or _Endpoints: Send to <partner name>_ for your partners.
+
+. If you can’t remember what to search for, scan the list for any endpoints that don’t have a message flow associated with them. These are indicated by a 0 in the message flow column.
```

Comment:
Limit steps to one sentence max next to the step number. Move other text below.
```suggestion
. If you can’t remember what to search for, scan the list for any endpoints that don’t have a message flow associated with them. 
These are indicated by a 0 in the message flow column.
```
This is a curious step. It is part of the step before it, but it IS a separate user action....
In any case, I am sure the audience will not be confused.

Reply:
It's a bad UI design, thus the steps are clunky.